akka-http-jwt/src/main/scala/akka/http/jwt/scaladsl/JwtSettings.scala

```diff
@@ -0,0 +1,19 @@
+package akka.http.jwt.scaladsl
+
+import akka.actor.ClassicActorSystemProvider
+import akka.annotation.{ ApiMayChange, DoNotInherit }
+import akka.http.jwt.impl.settings.{ JwtSettingsImpl, JwtSupport }
+import com.typesafe.config.Config
+
+@ApiMayChange @DoNotInherit
+trait JwtSettings {
+  def jwtSupport: JwtSupport
```

Comment:
Since the actual type is internal:

```suggestion
  /** INTERNAL API */
  @InternalApi private[akka] def jwtSupport: JwtSupport
```